content/projects/jcasc/dev-tools.adoc

```diff
@@ -0,0 +1,59 @@
+---
+layout: gsocproject
+title: "Dev-Tools for JCasC"
+goal: "IDE integrations for JCasC, along with a better testing architecture for the JSON schema, and addition of configuration extension points"
+category: Plugin
+year: 2019
+status: Active
+showGoogleDoc: false
+student: sladyn98
+mentors:
+- "timja"
+- "casz"
+links:
+  gitter: https://gitter.im/jenkinsci/jcasc-devtools-project
+  draft: https://docs.google.com/document/d/1aPfkmyMQRCcipVa0htFt-i7X_pkQHSVKfR5qT60HQZ8/edit?usp=sharing
+---
```

Comment:
For better social media engagement, could we include an opengraph image reference for configuration as code?
```suggestion
opengraph:
  image: /images/logos/JCasC/JCasC.png
---
```

Reply:
Definitely